src/plugins/data/common/query/timefilter/types.ts

```diff
@@ -20,14 +20,15 @@
 import { Moment } from 'moment';
 
 export interface RefreshInterval {
+  display?: string;
   pause: boolean;
   value: number;
 }
 
 export interface TimeRange {
   from: string;
   to: string;
-  mode?: 'absolute' | 'relative';
+  mode?: 'absolute' | 'relative' | 'quick';
```

Comment:
@qn895 I'm slightly confused on how both these fields get used.

I don't even see any place in the code using them (nor here or in the data plugin). 

:raised_eyebrow:  

I'd love to sync on this

Reply:
@lizozom Thanks for the feedback. Our team decided it would be good to take those parameters out as they are outdated. I have reverted the additions to the data plugin here: https://github.com/elastic/kibana/pull/75696/commits/5eefb05b65a11c4bb52098e4e99b2914d44aca8e